types/gulp-postcss/gulp-postcss-test.ts

```diff
@@ -0,0 +1,43 @@
+import * as gulp from 'gulp';
+import * as postcss from 'gulp-postcss';
+import * as autoprefixer from 'autoprefixer';
+import * as Vinyl from 'vinyl'
```

Comment:
How I should get this packages for your project? I did not see some dependencies in `package.json` for testing of other packages...